src/components/PlaceIcon.jsx

```diff
@@ -2,17 +2,43 @@ import React from 'react';
 import IconManager from 'src/adapters/icon_manager';
 import { getLightBackground } from 'src/libs/colors';
 import classnames from 'classnames';
+import { Heart } from 'src/components/ui/icons';
 
-const PlaceIcon = ({ place, withBackground }) => {
-  const { iconClass, color, icomoon } = IconManager.get(place);
+const favoriteColor = '#cd1690';
+
+const PlaceIcon = ({ place, withBackground, className, isFavorite = false, ...rest }) => {
+  const { iconClass, color, icomoon } = place // Place should be a poi
+    ? IconManager.get(place)
+    : rest;
+
+  if (isFavorite) {
+    return <FavoriteIcon className={className} />;
+  }
 
   return <div
-    className={classnames('placeIcon', `icon-${iconClass}`, { icon: !icomoon })}
+    className={classnames('placeIcon', `icon-${iconClass}`, { icon: !icomoon }, className)}
     style={{
       color,
       backgroundColor: withBackground ? getLightBackground(color) : null,
     }}
   />;
 };
 
+const FavoriteIcon = ({ className }) => {
+  return (
+    <div
+      className={classnames('placeIcon', className)}
+      style={{
+        color: favoriteColor,
+        backgroundColor: getLightBackground(favoriteColor),
```

Comment:
The design team will probably notice it's not exactly the specified color, maybe it's worth declaring it already (`#fbd9ef`, aka. "pink-lighter").